content/en/network_monitoring/devices/guide/build-ndm-profile.md

```diff
@@ -71,16 +72,165 @@ metrics:
 
 **Note**: `sysobjectid` can be a wildcard pattern to match a sub-tree of devices, for example: `1.3.6.1.131.12.4.*`.
 
-## Test the profile
+### Generate a profile file from a collection of MIBs
```

Comment:
So it feels like this whole new section could actually be steps - with what you input as the options and then what it outputs as the results, but I am not completely positive because I am not familiar enough with how to actually do what is in the rest of the guide. It just feels like it is missing context that will become clear if it is in steps.

Reply:
Reorganized this section to be more streamlined. @apigirl Ready for final review! 🎉